packages/block-library/src/image/edit.native.js

```diff
@@ -430,6 +434,62 @@ export class ImageEdit extends Component {
 		this.onSetSizeSlug( newValue );
 	}
 
+	onSetFeatured( mediaId ) {
+		const { closeSettingsBottomSheet } = this.props;
+		setFeaturedImage( mediaId );
+		closeSettingsBottomSheet();
+	}
+
+	getSetFeaturedButton( isFeaturedImage ) {
+		const { attributes, getStylesFromColorScheme } = this.props;
+
+		// eslint-disable-next-line no-unused-vars
+		const androidOnly = Platform.OS === 'android';
+
+		const featuredButtonStyle = getStylesFromColorScheme(
+			styles.featuredButton,
+			styles.featuredButtonDark
+		);
+
+		const setFeaturedButtonStyle = getStylesFromColorScheme(
+			styles.setFeaturedButton,
+			styles.setFeaturedButtonDark
+		);
+
+		return (
+			<>
+				{
+					// eslint-disable-next-line no-undef
+					androidOnly && (
+						<PanelBody>
+							{ isFeaturedImage ? (
+								<BottomSheet.Cell
```

Comment:
Nit, and mostly a personal pref but, I wonder what do you think about defining a couple of consts to hold the BottomSheet.Cells and use them in the ternary? I feel like it's a bit too nested and idented the way it currently is, making reading the code more difficult. Again, this comes down to personal preference mostly so, feel free to keep as is.

Reply:
I like that idea and agree it'd make the code a bit more readable. :) I've gone ahead to commit it.